### ISSUE TITLE
Disable Widevine in Tor mode

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#1076]
Carried over from https://github.com/brave/browser-laptop/issues/13626

**Comment by kjozwiak** [comment]
Still an issue, easiest way to reproduce, see the issue occurring:

* visit https://bitmovin.com/demos/drm

You'll notice that you'll get a Widevine notification and can install it inside a Tor Window. Example:

![installingwidevinetor](https://user-images.githubusercontent.com/2602313/53543492-dc0e5680-3af0-11e9-8b7f-212c33a22b62.gif)

**Comment by iefremov** [comment]
Blocking the Widevine installation prompt is trivial, we have just to add a check for Tor profile in `BraveDrmTabHelper::OnWidevineKeySystemAccessRequest`. Do we also have to block playing Widevine videos in Tor windows if Widevine is already installed? @diracdeltas @bsclifton

**Comment by diracdeltas** [comment]
i think we should not allow widevine in tor mode even if already installed (same with flash if its installed) but defer to @tomlowenthal

**Comment by tomlowenthal** [comment]
I agree with @diracdeltas on this one. My preference is to disable Widevine entirely in Tor: no prompt to install, and no playing if it's already installed.

**Comment by iefremov** [comment]
No playing goal should be achievable by disabling the content setting for plugins.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
